README.md

```diff
@@ -9,4 +9,11 @@ Requires >= Node v20
 2. run 'npm run build' to compile
 3. run 'npm run start' to start the service
 
-use localhost:3000/limited-partnerships/start to view the test page
\ No newline at end of file
+use localhost:3000/limited-partnerships/start to view the test page
+
+## Frontend Technologies and Utils
+
+- [NodeJS](https://nodejs.org/)
```

Comment:
I assume we'll add to this as we go along, docker instructions etc.

Reply:
Yes, I think whenever there's a commit and something needs documenting in the README related to it, that's the time to update the file.